Documentation/doc/Documentation/Installation.txt

```diff
@@ -1162,9 +1162,9 @@ environment variable. The environment variable `TBB_ARCH_PLATFORM=<arch>/<compil
 `<arch>` is `ia32` or `intel64`. `<compiler>` describes the Linux kernel, gcc version or Visual Studio version
 used. It should be set to what is used in `$TBBROOT/lib/<arch>`.
 
-The folder `TBBROOT/bin/<arch>/<compiler>` should be added to the `PATH` variable.
+The folder `TBBROOT/lib/<arch>/<compiler>` should be added to the `PATH` variable.
```

Comment:
That does not make any sense! Nothing from `TBBROOT/lib` should be in the `PATH`. On Windows, the DLL (the runtime part of shared libraries) are in `TBBROOT/bin`, and DLLs are searched in the `PATH`. On non-Windows, as far as I know there is nothing to add in the `PATH`.